core/app/models/spree/order.rb

```diff
@@ -27,7 +27,7 @@ class CannotRebuildShipments < StandardError; end
       go_to_state :confirm
     end
 
-    self.whitelisted_ransackable_associations = %w[shipments user promotions bill_address ship_address line_items]
+    self.whitelisted_ransackable_associations = %w[shipments user order_promotions promotions bill_address ship_address line_items]
```

Comment:
Do we want to keep `promotions` here? I guess for the separate Promo page's search?